src/Nervos/HeaderDep.h

```diff
@@ -0,0 +1,19 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <array>
+#include <vector>
+
+namespace TW::Nervos {
+
+using HeaderDep = Data;
+
+/// A list of header deps
+class HeaderDeps : public std::vector<HeaderDep> {};
```

Comment:
do this really need a class? just using std::vector or type defines

Reply:
Used using now.